storage/innobase/xtrabackup/src/fil_cur.cc

```diff
@@ -57,20 +57,7 @@ static bool xb_get_zip_size(const char *file_name, pfs_os_file_t file,
     page_size.copy_from(page_size_t(flags));
   }
 
-  if (page_size.is_compressed() || FSP_FLAGS_GET_ENCRYPTION(flags)) {
-    is_compressable = false;
-  } else {
-    const auto ret = os_file_read(read_request, file_name, file, buf, 0,
-                                  page_size.physical() * 2);
-    if (!ret) {
-      return (false);
-    }
-    if (Compression::is_compressed_page(buf + page_size.physical())) {
-      is_compressable = false;
-    } else {
-      is_compressable = true;
-    }
-  }
+  is_compressable = FSP_FLAGS_GET_ENCRYPTION(flags) ? false : true;
```

Comment:
with so many compressions around, would be nice to document what this variable is for
1. key_block_size=, record level compression
2. compression='zlib' page level IO compression
3. backup compression, --compress

:)

Reply:
Adjusted comment on the header file we declare is_compressable.

Reply:
is it committed?